test/test-agent-stopped.js

```diff
@@ -21,9 +21,13 @@ var http = require('http');
 
 describe('test-agent-stopped', function() {
   var agent;
-  before(function() {
+  before(function(done) {
```

Comment:
These changes will be obsolete after https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/410